Movies_app/Movies_app/Coordinator/ApplicationCoordinator.swift

```diff
@@ -0,0 +1,30 @@
+// ApplicationCoordinator.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+///
+final class ApplicationCoordinator: BaseCoordinator {
+    // MARK: - Public properties
+
+    override func start() {
+        guard let assemblyModuleBuilder = assemblyModuleBuilder else { return }
+        toMain(assemblyModuleBuilder: assemblyModuleBuilder)
+    }
```

Comment:
```suggestion
    override func start() {
        guard let assemblyModuleBuilder = assemblyModuleBuilder else { return }
        toMain(assemblyModuleBuilder: assemblyModuleBuilder)
    }
```
енто метод спусти ниже